functions/strings.js

```diff
@@ -0,0 +1,367 @@
+module.exports = {
```

Comment:
please don't add this file here. it should be in `src`

Reply:
This is leftover code from before you moved the files to `/src`. I'll move it now.